cmd/eskip/load.go

```diff
@@ -77,6 +77,17 @@ func loadRoutesChecked(in *medium) ([]*eskip.Route, error) {
 	return lr.routes, checkParseErrors(lr)
 }
 
+func checkRepeatedRouteIds(routes []*eskip.Route) error {
+	ids := map[string]struct{}{}
+	for _, route := range routes {
+		if _, ok := ids[route.Id]; ok {
+			return errors.New("Repeating route with id " + route.Id)
+		}
+		ids[route.Id] = struct{}{}
+	}
+	return nil
+}
+
```

Comment:
if you use `map[string]bool`, then the if check could simply look like: `if ids[route.Id] {`